typescript/tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "moduleResolution": "node",
     "baseUrl": ".",
     "paths": {
-      "ts-typescript-system-types": ["interfaces/ts-typescript-system-types.d.ts"]
+      "ts:typescript-system/types": ["interfaces/ts-typescript-system-types.d.ts"]
```

Comment:
And then here in tsconfig.json, you can say where to find the generated type definition for it. The types are generated by `jco types`.